src/Collector/Stack.php

```diff
@@ -81,6 +81,11 @@ final class Stack
      */
     private $responseCode;
 
+    /**
+     * @var string
```

Comment:
We can use type declarations in the methods below, can't we?

Reply:
yep. still should keep them on properties i think (at least that has been the best practice we followed so far)

Reply:
but this is `string|null` i think, its not always set.

Reply:
Yes, it is. There was no `null` specified on the other properties, so I wasn't sure whether they are always set or if `null`-able is just not marked. I will update the hint.

Reply:
annotations seem to be a bit sloppy here. i'd make the new ones correct nonetheless. when you do the return type declaration, php would explode when that one is not correct, and having the property itself correct will help us to see what it needs to be.